docs/knowledgebase/runtime/storage.md

```diff
@@ -305,25 +285,25 @@ GenesisConfig {
 }
 ```
 
-#### `build`
+### `build`
 
 Whereas [the `config` extension](#config) to the `decl_storage` macro allows you to configure a
 module's genesis storage state within a chain specification, the `build` extension allows you to
-perform this same task within the module itself (this gives you access to the module's private
+perform this same task within the module itself (which gives you access to the module's private
 functions). Like `config`, the `build` extension accepts a single parameter, but in this case the
 parameter is always required and must be a closure, which is essentially a function. The `build`
-closure will be invoked with a single parameter whose type will be the pallet's `GenesisConfig` type
-(this gives you easy access to all the attributes of the `GenesisConfig` type). You may use the
-`build` extension along with the `config` extension for a single storage item; in this case, the
-pallet's `GenesisConfig` type will have an attribute that corresponds to what was set using `config`
+closure will be invoked with a single parameter whose type will be the module's `GenesisConfig` type. This gives you easy access to all the attributes of the `GenesisConfig` type. 
```

Comment:
I see that at some places you replaced the word "module" to "pallet", and other places you are deliberately changing "pallet" back to "module". What would be a good guiding principle when to use "pallet" and when to use "module"?

Reply:
Yes, this came up a lot and does need fixing. However, I decided to focus this PR on re-structuring rather than making terminology changes. I suggest we leave as is for this PR and I'll do another pass on making sure all occurrences of the terms "pallet" and "module" are consistent with their contexts. My thoughts are to move away from the term "module" completely, unless it's referring to a Rust (or non-FRAME) thing.